docs/docs/configuration/object_detectors.md

```diff
@@ -140,6 +140,22 @@ The OpenVINO device to be used is specified using the `"device"` attribute accor
 
 OpenVINO is supported on 6th Gen Intel platforms (Skylake) and newer. It will also run on AMD CPUs despite having no official support for it. A supported Intel platform is required to use the `GPU` device with OpenVINO. For detailed system requirements, see [OpenVINO System Requirements](https://docs.openvino.ai/2024/about-openvino/release-notes-openvino/system-requirements.html)
 
+:::tip
+
+Whening using many cameras one detector may not be enough to keep up. Multiple detectors can be defined assuming GPU resources are available. An example configuration would be:
```

Comment:
Typo here: “whening”

Reply:
Thanks! just so you know, you can use the icon with +- in the formatting bar to make a suggestion so you can get credit for it 👍